monitoring/osd-managing-metrics.adoc

```diff
@@ -0,0 +1,38 @@
+[id="managing-metrics"]
+= Managing metrics
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: managing-metrics
+
+toc::[]
+
+This section contains examples that show how to manage your metrics.
```

Comment:
You could expand this introduction because the assembly does a lot more than this. For example, here is the TOC (which I copied from https://docs.openshift.com/container-platform/4.7/monitoring/managing-metrics.html):

    Understanding metrics
    Setting up metrics collection for user-defined projects
        Deploying a sample service
        Specifying how a service is monitored
    Querying metrics
        Querying metrics for all projects as a cluster administrator
        Querying metrics for user-defined projects as a developer
        Exploring the visualized metrics
    Next steps

You could summarise this as follows, for example:

"This document provides an overview about how {product-title} are collected, queried and visualized."

Reply:
Done, thanks.